content/zmc-challenges/first-website/html-on-phone-or-computer/_index.md

```diff
@@ -1,58 +1,59 @@
 content_type: topic
 title: How to get set up to author web code on your own device
 ---
 
-In this step you'll be learning about how to write HTML files on your own computer or cellphone. As nice as the W3Schools Try It Yourself editor is, it's not a professional web development tool and you'll need something more powerful if you want to make a proper website. 
+In this step you'll be learning about how to write HTML files on your own computer or cellphone. As nice as the W3Schools Try It Yourself editor is, it's not a professional web development tool and you'll need something more powerful if you want to make a proper website.
 
-Basically what we are going to do here is: Make use of a **text editor** to write **HTML files** that can be opened in a **web browser**. 
+Basically what we are going to do here is: Make use of a **text editor** to write **HTML files** that can be opened in a **web browser**.
 
 If you are not super sure what a file is then please read [this](https://www.howtogeek.com/757092/what-are-computer-files-and-folders/) before you continue. If you are still not sure, then please ask on the Discord channel.
 
-The first thing you need to do here is choose a text editor to work with. There are a few different options, please choose the one that works best for you. 
+The first thing you need to do here is choose a text editor to work with. There are a few different options, please choose the one that works best for you.
 
-## Option 1: Just use a text editor on your computer 
+## Option 1: Just use a text editor on your computer
 
-If you are using a computer and want to get set up in the easiest possible way then you can just use the built-in text editor. 
+If you are using a computer and want to get set up in the easiest possible way then you can just use the built-in text editor.
 
 - If you are running Windows you can just use Notepad. [Here](https://www.youtube.com/watch?v=0BA5Q4RS4xo) is a video demonstration.
 - If you are running a Mac then you can use TextEdit.
 
 You can see more detailed instructions [here](https://www.w3schools.com/html/html_editors.asp)
 
-## Option 2: Use your Android phone
+## Option 2: Use your smartphone
 
-If you have an Android then you can make use of the [ACode](https://play.google.com/store/apps/details?id=com.foxdebug.acodefree&hl=en_ZA) code editor.
+- If you have an Android then you can make use of the [ACode](https://play.google.com/store/apps/details?id=com.foxdebug.acodefree&hl=en_ZA) code editor. ACode can be a little confusing. [Here](https://youtu.be/ivJ4FGKnGUE) is a little demonstration video that might help you.
 
-ACode can be a little confusing. [Here](https://youtu.be/ivJ4FGKnGUE) is a little demonstration video that might help you.
+- For an iOS device like an iPhone or iPad, you can use the [Koder](https://apps.apple.com/za/app/koder-code-editor/id1447489375) code editor. To learn how to use it, watch this [video](https://www.youtube.com/watch?v=U8obGsKFRCE)
```

Comment:
Our content strategy does allow us to make use of other resources as you have done here! Nice!

The problem is that there are a few bad practices in this video.

1. The file naming used here is not consistent with the ones we recommend that the students use.
2. The Syntax here is incorrect. The `head` tag is not closed properly.

Can you please try to look for another resource explaining Koder? If not, I will ask Sheena if this is sufficient